PolyEngine/Engine/Src/Engine.cpp

```diff
@@ -44,7 +44,7 @@ void Poly::Engine::Init(std::unique_ptr<IGame> game, std::unique_ptr<IRenderingD
 	RegisterWorldComponent<DeferredTaskWorldComponent>((size_t)eEngineWorldComponents::DEFERRED_TASK);
 	RegisterWorldComponent<SoundWorldComponent>((size_t) eEngineWorldComponents::SOUND);
 	RegisterWorldComponent<Physics2DWorldComponent>((size_t)eEngineWorldComponents::PHYSICS_2D);
-	RegisterWorldComponent<DiffuseLightSourceWorldComponent>((size_t)eEngineWorldComponents::DIFFUSE_LIGHTSOURCE);
+	RegisterWorldComponent<AmbientLightWorldComponent>((size_t)eEngineWorldComponents::DIFFUSE_LIGHTSOURCE);
```

Comment:
If we're changing class names, let's change enum field names as well.

Reply:
done